### ISSUE TITLE
[VMAccess] install fails on Oracle 7.2

### ISSUE THREAD

**Issue by hglkrijger** [Azure/azure-linux-extensions#197]
Installing VMAccess 1.4.0.0 via agent rc3 bits on Oracle Linux 7.2:

[root@o72rc3 Microsoft.OSTCExtensions.VMAccessForLinux-1.4.0.0]# ./vmaccess.py -install
Traceback (most recent call last):
  File "./vmaccess.py", line 36, in <module>
    waagent = load_waagent()
  File "/var/lib/waagent/Microsoft.OSTCExtensions.VMAccessForLinux-1.4.0.0/waagentloader.py", line 33, in load_waagent
    waagent.MyDistro = waagent.GetMyDistro()
  File "/var/lib/waagent/Microsoft.OSTCExtensions.VMAccessForLinux-1.4.0.0/waagent", line 5840, in GetMyDistro
    return globals()[dist_class_name]() # the distro class inside this module.
  File "/usr/sbin/waagent2.0", line 1107, in __init__
    super(oracleDistro, self).__init__()
  File "/usr/sbin/waagent2.0", line 1014, in __init__
    super(redhatDistro,self).__init__()
TypeError: super(type, obj): obj must be an instance or subtype of type

**Comment by rginjupalli** [comment]
I notice the same error on OL 6.x as well

**Comment by boumenot** [comment]
This should be fixed with PR #208.

I used Oracle 7.2 for all of my testing because Oracle Linux was high on the list of failures for this extension.  I created test cases from [documentation][1], and with the fix they passed.

This was deployed as version 1.4.1.0, which means all 1.4 users automatically picked it up.  The issue still exists in the older version 1.0, 1.1, etc., but our data shows that the usage is very low for previous versions.  My plan is to remove those versions in the near future.

[1]: https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-linux-using-vmaccess-extension/
